scripts/json_parser.py

```diff
@@ -32,7 +32,7 @@ def fix_and_parse_json(
 ) -> Union[str, Dict[Any, Any]]:
     """Fix and parse JSON string"""
     try:
-        json_str = json_str.replace('\t', '')
+        json_str = json_str.replace('\t', "")
```

Comment:
These two statements are functionally the same and produce the same result. tabs are removed.